style.css/footerStyle.css

```diff
@@ -0,0 +1,19 @@
+.site-footer {
+  background-color: #072bce;
+  color: #ffffff;
+  text-align: center;
+  flex-shrink: 0;
```

Comment:
Could you please explain what flex shrink do here?

Reply:
That was extra,I removed it.